extension/chrome/settings/modules/my_key.ts

```diff
@@ -76,7 +76,8 @@ View.run(class MyKeyView extends View {
     try {
       const result = await this.pubLookup.attester.lookupEmail(this.acctEmail);
       const url = FlowCryptWebsite.url('pubkey', this.acctEmail);
-      if (result.pubkey && (await KeyUtil.parse(result.pubkey)).id === this.keyInfo.fingerprints[0]) {
+      const doesContainKey = await asyncSome(result.pubkeys, async (pubkey) => (await KeyUtil.parse(pubkey)).id === this.keyInfo.fingerprints[0]);
```

Comment:
We should compare keys using the `identityEquals` helper method, as ids (or fingerprints) are not guaranteed to be unique across key families. Here this line would fit good:
```
const doesContainKey = await asyncSome(result.pubkeys, async (pubkey) => KeyUtil.identityEquals(await KeyUtil.parse(pubkey), this.keyInfo));
```